### DISCUSSION TITLE
v-model does not bind correctly

### DISCUSSION THREAD

**Discussion by twiese27** [dm4t2/vue-currency-input#406] [Q&A]
Hi,
i have a laravel application with Vue. 
I have laravel version 7.30.4, Vue Version 2.6.14 and Vue Currency Input Version 2.5.1
Test.Vue:
```
<template>
	<CurrencyInput
		v-model="value"
		:options="{ currency: 'EUR' }"
	/>
</template>

<script>
import CurrencyInput from './../General/CurrencyInput.vue'

export default {
	name: 'App',
	components: { CurrencyInput },
	data: () => ({ value: 1234 })
}
</script>
```

CurrencyInput.Vue:

```
<template>
	<input
		ref="inputRef"
		type="text"
	/>
</template>

<script>
import { useCurrencyInput } from 'vue-currency-input'

export default {
	name: 'CurrencyInput',
	props: {
		modelValue: Number, // Vue 2: value
		options: Object
	},
	setup(props) {
		const { inputRef } = useCurrencyInput(props.options)

		return { inputRef }
	}
}
</script>
```

The Input Validation works and the numbers are correctly formatted after i type in. But it doesn't bind the value in the v-model. When the page is loaded value = null and not 1234 and 1234 is not shown at the input.
Can you help me?
![currency input](https://github.com/dm4t2/vue-currency-input/assets/81928647/018a67fc-3135-4481-8fef-fbfddf2fdb41)

**Comment by dm4t2** [comment]
You are using the Vue 3 Syntax for the v-model prop (`modelValue`). It should be `value` (like described in the inline code comment).
